src/Features/CSharp/Portable/Completion/KeywordRecommenders/CheckedKeywordRecommender.cs

```diff
@@ -18,10 +21,49 @@ public CheckedKeywordRecommender()
 
         protected override bool IsValidContext(int position, CSharpSyntaxContext context, CancellationToken cancellationToken)
         {
-            return
-                context.IsStatementContext ||
+            if (context.IsStatementContext ||
                 context.IsGlobalStatementContext ||
-                context.IsNonAttributeExpressionContext;
+                context.IsNonAttributeExpressionContext)
+            {
+                return true;
+            }
+
+            var targetToken = context.TargetToken;
+
+            if (targetToken.Kind() == SyntaxKind.OperatorKeyword)
+            {
+                var previousPossiblySkippedToken = targetToken.GetPreviousToken(includeSkipped: true);
+
+                if (previousPossiblySkippedToken.IsLastTokenOfNode<TypeSyntax>())
+                {
+                    return true;
+                }
+
+                SyntaxToken previousToken;
+
+                if (previousPossiblySkippedToken.IsLastTokenOfNode<ExplicitInterfaceSpecifierSyntax>())
+                {
+                    var firstSpecifierToken = previousPossiblySkippedToken.GetAncestor<ExplicitInterfaceSpecifierSyntax>().GetFirstToken(includeSkipped: true);
+
+                    if (firstSpecifierToken.GetPreviousToken(includeSkipped: true).IsLastTokenOfNode<TypeSyntax>())
+                    {
+                        return true;
```

Comment:
so 'checked' is allowed, in an explicit-impl, even without having 'explicit' in the sig?

Reply:
> so 'checked' is allowed, in an explicit-impl, even without having 'explicit' in the sig? 

An 'explicit' keyword has nothing to do with explicit interface implementation, it is part of a conversion operator declaration

Reply:
sorry, what i meant here is that this code checks if we're after an explicit-interface specifier, and then checks that right before that is a type-syntax.  But it does *not* do the `previousToken.Kind() == SyntaxKind.ExplicitKeyword` check below.  I'm trying to understand if that's ok and desirable behavior.  I commented on the test that seems to be hitting this, and i was trying to understand if this is a legal language construct and what it means.

Reply:
A conversion operator cannot have a type between 'explicit' and 'operator', its type goes before parameters. For example:
```
public static explicit operator checked TypeToConvertTo(TypeToConvertFrom x) {}
```